manifest.json

```diff
@@ -669,6 +669,28 @@
               "variant": "v8"
             }
           ]
+        },
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/windows-nano/1803/helix/amd64",
+              "os": "windows",
+              "tags": {
+                "nanoserver-1803-amd64-helix-$(System:DockerfileGitCommitSha)-$(System:TimeStamp)": {}
```

Comment:
amd64 and helix should be swapped in the tag name.

Reply:
done